packages/sw-runtime-caching/src/lib/stale-while-revalidate.js

```diff
@@ -24,6 +24,10 @@ import assert from '../../../../lib/assert';
  *
  * In addition to updating the appropriate caches, it will also trigger any
  * appropriate plugins defined in the underlying `RequestWrapper`.
+ * 
+ * This class always returns the cache and returns it first regardless of how
```

Comment:
> "This class always returns the cache and returns it **first** regardless of how quickly a network response returns."

This phrasing isn't accurate—"returns it first" implies that the network response is also going to be returned in addition to the previously cached response, which isn't the case.

The network response is used to update the cache asynchronously, after the previously cached response is returned to fulfill the request.

If there wasn't a previously cached response, then the network response **is** returned to the client in addition to being used to update the cache, but in that scenario, there's also only a single response returned to the client, so the "first" phrasing is still misleading.